src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "CSS backdrop-filter"
+subhead: |
+  Enabled in Chrome 76
+date: 2019-06-25
+authors:
+  - adamargyle
+hero: hero.jpeg
+alt: Women behind frosted glass
+description: |
+  CSS backdrop-filter was made famous by Apple's design team. It brings
+  the ability for a background to apply a filter to the contents beneath.
+  Creates a nice layering effect that keeps beneath elements in context.
+tags:
+  - post
+  - CSS
+  - backdrop-filter
+---
+
+Designers and developers alike can celebrate as a long awaited CSS feature [releases in Chrome 76](https://www.chromestatus.com/feature/5679432723333120): backdrop-filter. Frosted glass effects, video overlays, translucent navigation headers, inappropriate image censoring, image loading, and more, no longer require [hacks or workarounds](https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter).
```

Comment:
i like a lot of it, but the ending isn't there yet. it's not a new css property, chrome is late to this game by a long shot. safari's had it since like 2015 or something, and edge since 2018. i was pretty intentionally leaving it out how far behind Chrome is. 

i agree the ", and more, no longer.." is an awkward transition. needs improvement.

Reply:
intro paragraph updated:

> Designers and developers alike can celebrate as a long awaited CSS feature [arrives in Chrome 76](https://www.chromestatus.com/feature/5679432723333120): `backdrop-filter`. Until now, [hacks or workarounds](https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter) were required to create the effects that backdrop-filter provides: frosted glass, video overlays, translucent navigation headers, inappropriate image censoring, image loading, etc.